packages/experimental/Switch/src/useSwitch.ts

```diff
@@ -35,7 +35,7 @@ export const useSwitch = (props: SwitchProps): SwitchInfo => {
   const onKeyUpProps = useKeyProps(toggleCallback, ' ', 'Enter');
 
   // Triggers animation only when the checked state changes
-  if (prevCheckedState !== checkedState) {
+  if (prevCheckedState !== checkedState && AccessibilityInfo.isReduceMotionEnabled) {
```

Comment:
Have you tested this? Does it work? Just wondering if there's any native work required for win32 to make `isReduceMotionEnabled` return the right value according to the OS settings.

Reply:
Yeah I tried it out and it worked. I went toggled this "Show animations in windows" setting off.
![image](https://user-images.githubusercontent.com/20259210/182467680-23a91b39-1aeb-4515-b548-ec623706d70e.png)

Reply:
Okay, good to know!